src/services/ai-server/ai-server/ai.server.service.ts

```diff
@@ -170,17 +171,22 @@ export class AiServerService {
       LogContext.AI_SERVER
     );
     this.eventBus.publish(
-      new IngestSpace(
+      new IngestBodyOfKnowledge(
         persona.bodyOfKnowledgeID,
-        SpaceIngestionPurpose.KNOWLEDGE,
+        persona.bodyOfKnowledgeType,
+        IngestionPurpose.KNOWLEDGE,
         persona.id
       )
     );
   }
 
   public async ensureContextIsIngested(spaceID: string): Promise<void> {
```

Comment:
misleading name IMO - It only publishes an event. Discard if you think otherwise